reference/javascript_3x/APIReference.md

```diff
@@ -644,6 +644,28 @@ mediaObject[ADB.Media.MediaObjectKey.MediaResumed] = true;
 tracker.trackSessionStart(mediaObject);
 ```
 
+#### Granular ad tracking
+
+Constant to enable granular (every second) pings for ad content. This information must be provided when starting a tracking session.
```

Comment:
should we mention by default, it is using 10 sec pings for Ads?

Reply:
@cacheung Added.